hazelcast/src/test/java/com/hazelcast/spi/impl/operationexecutor/slowoperationdetector/SlowOperationDetectorBasicTest.java

```diff
@@ -46,6 +46,21 @@ public void teardown() {
     }
 
     @Test
+    public void testDisabled() {
+        Config config = new Config();
+        config.setProperty(GroupProperties.PROP_SLOW_OPERATION_DETECTOR_ENABLED, "false");
+        config.setProperty(GroupProperties.PROP_SLOW_OPERATION_DETECTOR_THRESHOLD_MILLIS, "1000");
+
+        instance = getSingleNodeCluster(config);
```

Comment:
Why don't you make use of the HazelcastTestSupport.createHazelcastInstance method?